tcms/testplans/static/testplans/js/get.js

```diff
@@ -358,6 +358,31 @@ function toolbarEvents() {
         }
 
     });
+
+    $('#default-tester-button').click(function(ev) {
+        let selectedCases = getSelectedTestCases();
+
+        if (!selectedCases.length) {
+            alert($('#test_plan_pk').data('trans-no-testcases-selected'));
+            return;
+        }
+
+        var email_or_username = window.prompt($('#test_plan_pk').data('trans-default-tester-prompt-message'));
+        if (!email_or_username) {
+            return;
+        }
+        for (let i = 0; i < selectedCases.length; i++) {
+            let testCaseId = selectedCases[i];
+
+            jsonRPC('TestCase.update', [testCaseId, {'default_tester': email_or_username}], function(tc) {
+                const testCaseRow = $(`[data-testcase-pk=${testCaseId}]`);
+                animate(testCaseRow, function() {
+                    testCaseRow.find('.js-test-case-tester').html(tc.default_tester);
```

Comment:
this should render the entire row instead b/c that would be required in other places and we should start making the handling unified everywhere on this page.